### PR TITLE
Testing #3 homework

### PR DESCRIPTION
**Pull request by m-kop**

### CODE REVIEW

**Review by Kasperice — APPROVED** [review]
Nice code, I do not have anything to add :)

**Review by WBigaj — APPROVED** [review]
I have checked your code and it looks good 👍

File: test/PizzeriaTest.cpp [hunk 1]
```diff
@@ -52,16 +56,47 @@ TEST_F(PizzeriaTest, completeOrderWithStubPizza)
...
+    EXPECT_CALL(timerMock, sleep_for(minutes(9))).Times(1);
+
+    // When
+    auto orderId = pizzeria.makeOrder(pizzas);
+    auto price = pizzeria.calculatePrice(orderId);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
+
+    // Then
+    ASSERT_EQ(75, price);
```
Comment (ziobron) [inline test/PizzeriaTest.cpp:101]: you have some magic numbers. Use constexpr for associated values:
```cpp
constexpr double funghiPrice = 40.0;
...
EXPECT_CALL(funghiMock, getPrice()).WillOnce(Return(funghiPrice));
...
ASSERT_EQ(funghiPrice + margheritaPrice, price);
```

The same for baking time

**Review by ziobron — APPROVED** [review]

**Comment by ziobron** [comment]
{ "requirement_ids" : [1119, 1119, 1119, 3524, 3525, 3526] }

3 * 3 XP za punktualność (1119)
10 XP za TimeMock w Pizzas (3524)
15 XP za nowy przypadek testowy w Pizzas (3525)
5 XP za workflow w Pizzas (3526)

**Pull request closed without merging**